### PR TITLE
(maint) Fix spelling errors and typos in lib/puppet

### PR DESCRIPTION
**Pull request by gguillotte**
Fix low-priority errors in comments, strings, and error messages.

### CODE REVIEW

File: lib/puppet/face/certificate.rb [hunk 1]
```diff
@@ -74,7 +74,7 @@
       options[:dns_alt_names] = Puppet[:dns_alt_names] if global_setting_from_cli
 
       # If dns_alt_names are specified via the command line, we will always add
-      # them. Otherwise, they will default to the config file setting iff this
```
Comment (joshcooper) [inline lib/puppet/face/certificate.rb]: I think `iff` means `if and only if` in this context

**Comment by joshcooper** [comment]
@gguillotte unfortunately there are conflicts. Could you rebase and we'll merge?

**Comment by gguillotte** [comment]
@joshcooper Done!

**Comment by joshcooper** [comment]
The appveyor failure is due to PUP-6473, I'll close this and reopen, to pick up the change.

**Pull request merged**